Source/Frontend/UI/Components/Memory Tools/Common.cs

```diff
@@ -0,0 +1,27 @@
+namespace RTCV.UI
+{
+    using System.IO;
+    using System.Windows.Forms;
+
+    internal static class Common
+    {
+        internal static bool CopyFileWithOverwritePrompt(string sourcePath, string targetDirectory)
```

Comment:
It feels more idiomatic to throw an Exception if the user cancels the copy, rather than return a boolean. However, I'm not sure if that would be too verbose for this situation.

Reply:
I moved this to use `OverwriteCancelledException`